speech/cloud-client/transcribe_word_time_offsets.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Speech API sample that demonstrates word time offsets.
+
+Example usage:
+    python transcribe_word_time_offsets.py resources/audio.raw
+    python transcribe_word_time_offsets.py \
+        gs://cloud-samples-tests/speech/vr.flac
+"""
+
+import argparse
+import io
+
+
+def transcribe_file_with_word_time_offsets(speech_file):
+    """Transcribe the given audio file synchronously and output the word time
+    offsets."""
+    from google.cloud import speech
+    from google.cloud.speech import enums
+    from google.cloud.speech import types
+    client = speech.SpeechClient()
+
+    with io.open(speech_file, 'rb') as audio_file:
+        content = audio_file.read()
+
+    audio = types.RecognitionAudio(content=content)
+    config = types.RecognitionConfig(
+        encoding=enums.RecognitionConfig.AudioEncoding.LINEAR16,
+        sample_rate_hertz=16000,
+        language_code='en-US',
+        enable_word_time_offsets=True)
+
+    response = client.recognize(config, audio)
+
+    alternatives = response.results[0].alternatives
+
+    for alternative in alternatives:
+        print('Transcript: {}'.format(alternative.transcript))
+
+        for word_info in alternative.words:
+            word = word_info.word
+            start_time = word_info.start_time
+            end_time = word_info.end_time
+            print('Word: {}, start_time: {}, end_time: {}'.format(
+                word,
+                start_time.seconds + start_time.nanos * 1e-9,
+                end_time.seconds + end_time.nanos * 1e-9))
+
+
+# [START def_transcribe_gcs]
```

Comment:
The `transcribe_file_with_word_time_offsets` function doesn't use a region tag but this does. Ignore comment if that's how Python often does samples!

If using region tags, the GCS tag is: `speech_async_recognize_gcs_words`

Again, if you normally use `def_...` region tags, then all good!

---

I'm guessing that we should embed these snippets using indented region for  `transcribe_file_with_word_time_offsets`  but region rag for `def_transcribe_gcs`?